app/config.js

```diff
@@ -9,6 +9,6 @@ angular.module("ngApp.config", [])
     "auth_api": "__AUTH__/api",
     "supportedAPI": "1",
     "tag": "https://github.com/NCI-GDC/portal-ui/releases/tag/__VERSION__",
-    "production": __PRODUCTION__,
-    "fake_auth": __FAKE_AUTH__,
+    "production": "__PRODUCTION__",
+    "fake_auth": "__FAKE_AUTH__",
```

Comment:
also replacing the quotes https://github.com/NCI-GDC/portal-ui/pull/984/files#diff-b9e12334e9eafd8341a6107dd98510c9R147